include/zephyr/net/ethernet.h

```diff
@@ -49,10 +49,12 @@ extern "C" {
  * @{
```

Comment:
Please change the subject of the mac address length commit to
`net: ethernet: add definition for mac address length`
so that it is easier to search for network related commits if needed.

Reply:
Sorry, forgot to adjust it at the end, done now, thanks.